### PR TITLE
product card was added

### PR DESCRIPTION
**Pull request by Yuliia-Hrinchenko1**
- [DEMO LINK](https://Yuliia-Hrinchenko1.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://Yuliia-Hrinchenko1.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,14 @@
+@import "../styles/reset.css";
+@import "../styles/blocks/card.css";
+@import "../styles/blocks/star.css";
+
+* {
+  box-sizing: border-box;
+}
```
Comment (Lilu580) [inline src/style.css:7]: i think this part of code should be in reset.css

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+html {
+  font-family: Roboto, sans-serif;
+  color: #060b35;
+  font-size: 10px;
+  line-height: 14px;
+}
```
Comment (Lilu580) [inline src/style.css:14]: I`m not sure, but i think that beter to create new file css for set syles in page.Like page.css and give your html-tag class="page".
```suggestion

.page {
  font-family: Roboto, sans-serif;
  color: #060b35;
  font-size: 10px;
  line-height: 14px;
}
```

File: styles/blocks/card.css [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+  font-size: 16px;
+  line-height: 18px;
+}
+
+.card__buy-button {
+  display: flex;
+  align-items: center;
+  height: 40px;
+  padding: 11px 69px;
+
```
Comment (Lilu580) [inline styles/blocks/card.css:58]: i think better way to center its 
```suggestion
  justify-content:center;
  text-align:center;
```

File: styles/blocks/card.css [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+  font-weight: 700;
+  font-size: 14px;
+  color: #fff;
+  text-transform: uppercase;
+
+  border: 1px solid transparent;
+  border-radius: 5px;
+  background-color: #00acdc;
```
Comment (Lilu580) [inline styles/blocks/card.css:66]: for color i think you should use variable,for easy changing your button color.Just create file variables.css and put here
```suggestion
  root:{
  --button-color:#00acdc;
```
and after that
```suggestion
  background-color: var(--button-color);
```

File: styles/blocks/card.css [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+  cursor: pointer;
+}
+
+.card__buy-button:hover {
+  color: #00acdc;
+  background-color: #fff;
+  border-color: #00acdc;
```
Comment (Lilu580) [inline styles/blocks/card.css:73]: same 
```suggestion
  background-color: var(--button-color);
```

File: styles/blocks/card.css (hunk 3 above)
Comment (Lilu580) [reply styles/blocks/card.css:58]: aha you have no parent container for your link,so i`m not sure about this

**Review by Lilu580 — COMMENTED** [review]
Reviewed

File: styles/page.css [hunk 6]
```diff
@@ -0,0 +1,10 @@
+* {
+  box-sizing: border-box;
+}
```
Comment (Lilu580) [inline styles/page.css:3]: Lets mentor decide where to place * page.css or reset.css

**Review by Lilu580 — COMMENTED** [review]
Question for mentor

**Review by Andr1yK — APPROVED** [review]

**Pull request closed without merging**